firefox/janitor-hg.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Firefox (hg)",
+  "description": "The Firefox browser (hg flavour)",
+  "icon": "https://janitor.technology/img/firefox.svg",
+  "docker": {
+    "image": "JanitorTechnology/firefox-hg:base"
+  },
+  "ports": {
+    "22": {
+      "proxy": "none"
+    },
+    "8088": {
+      "proxy": "https",
+      "label": "VNC"
+    },
+    "8089": {
+      "proxy": "https",
+      "label": "Cloud9"
+    }
+  },
+  "previewUrls": {
+    "VNC": "https://${HOSTNAME}/${CONTAINER}/8089/vnc.html"
+  },
+  "scripts": {
+    "./mach build": "./mach build",
+    "./mach build faster": "./mach build faster",
+    "./mach build binaries": "./mach build binaries",
+    "./mach run": {
+      "cmd": "./mach run",
+      "openPreview": "VNC"
```

Comment:
Actually, it opens a proxied container port in a preview panel or tab, so maybe "openPreview" does make more sense. Not sure.